CompRep.tex

```diff
@@ -1,6 +1,20 @@
 \documentclass[11pt]{article}
 \usepackage{graphicx}
 \usepackage{cite}
+\usepackage{xcolor}
+\usepackage{listings}
+
+\def\UrlBreaks{\do\/\do-}
+\lstset{language=[08]Fortran, 
```

Comment:
I think this line is causing latex compile issues on the linux desktop:
"
The following languages weren't loadable:
    ,fortran$08
"

Reply:
Works if we get rid of "[08]"

Reply:
Yes, removing